common/conf/env/PROD.properties

```diff
@@ -23,6 +23,7 @@ ajax.cors.origin=http://www.theguardian.com,https://www.theguardian.com,https://
 
 amp.host=amp.theguardian.com
 amp.scheme=https://
+amp.cors.origin=https://www.theguardian.com,https://amp.theguardian.com, https://cdn.ampproject.org
```

Comment:
Did Google/AmpProject confirm amp content would always be served from `https://cdn.ampproject.org`